JS/Call-Apply/apply.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<body>
+       
+<p>Usage Of apply()</p>
+
+
+<script src="apply().js"></script>
```

Comment:
Does sourcing a script file like this work? I am curious
  src="apply().js"

Reply:
sorry ...nope karthik it wont work...i forgot to change ...after i did in local ...